aks/README.md

```diff
@@ -1,9 +1,21 @@
-0. Create an Azure Service Principal
-  ```
-  az ad sp create-for-rbac
-  ```
-  Note the SP must have sufficient privileges, the created account must have at least contributor role. The SP may alternatively be created via the portal by creating an app
-  registration account as assigning the specific role. The data provided as output from this function will be required in the terraform variables.
+0. Prerequisites
+  * Create an Azure Service Principal
+    ```
+    az ad sp create-for-rbac
+    ```
+    Note the SP must have sufficient privileges, the created account must have at least Contributor and DNS Zone Contributor roles. The data provided as output from this function will be required in the terraform variables.
+
+  * Create a Resource Group
+    ```
+    az group create --name $(az_resource_group) --location $(location)
+    ```
+    This resource group will contain all resources of, and in support of your SUSE CAP cluster. Make note of the _name_ and _location_ which must be supplied in terraform variables.
+
+  * Create a DNS zone
+    ```
+    az network dns zone create -g $(az_resource_group) -n $(dns_zone_name)
+    ```
+    The DNS zone will host the cluster's DNS records in a specific domain name. Make note of the _name_ which must be supplied in terraform variables. See https://docs.microsoft.com/en-us/azure/dns/ for details on setting up a DNS zone.
 
```

Comment:
The end-user will also need to update their domain registrar's nameserver records with the name servers populated in the Azure DNS zone so that the domain registrar knows which NS would resolve the domain records.